app/controllers/users.js

```diff
@@ -0,0 +1,26 @@
+const usersService = require('../services/users'),
+  logger = require('../logger'),
+  errors = require('../errors');
+
+exports.create = (req, res, next) =>
+  usersService
+    .findByEmail(req.body.email, res)
```

Comment:
What happen if tomorrow you search by name or lastName? You pass condition. For example `{mail: req.body.email}`